src/foam/nanos/dig/DigSelectPropertyView.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.dig',
+  name: 'DigSelectPropertyView',
```

Comment:
We'll make a widget for this later. For this PR, we'll just use the default view that StringArray provides.